### ISSUE TITLE
Version number

### ISSUE THREAD

**Issue by peyre** [flathub/org.raceintospace.Raceintospace#2]
The version number on the main page identifies this as v.1.2, which I think is appropriate, since it's in between 1.1 and the not-yet-released 2.0.  However when you run the game, at the top it identifies itself as version 2.0.0rc1.  Could we get it to show as 1.2 at the top?

**Comment by hfiguiere** [comment]
The version on the flathub page is taken from the version the appstream file, which comes from upstream. So if there is a discrepency between that and the actual version upstream, it's a bug for them.

From the flathub point of view, the version should be displayed as 2.0.0rc1 (the appstream file would need to be patched)


(not sure what you mean by "at the top" though)

**Comment by peyre** [comment]
Hubert, thanks for getting back to me so soon!  "[A]t the top" meant in the title bar of the window while the game is running (see below).

Incidentally, user pemensik on our project asked me (some time ago) to make sure he had admin rights to this repository on Flathub, but I missed it until recently.  I came here to check it but don't see any option to manage users.  I'm not a programmer and am very new to Flathub, so I'm a little out of my depth.  Does he have admin rights to this project here?

![SplashScreen](https://user-images.githubusercontent.com/3118014/165994682-1b133108-9bd6-4601-8575-a0481abf6842.png)

**Comment by hfiguiere** [comment]
So the appdata should be fixed to have a `2.0.0rc` release, and it comes from upstream. (a patch can be made here in the mean time)

Sorry I don't have any power to send invite to repositories. @barthalion can however.

**Comment by barthalion** [comment]
I've just checked and @pemensik definitely has the write access here.

**Comment by peyre** [comment]
Thanks for checking!

**Comment by pemensik** [comment]
I have pushed an updated release with a recent version. But I guess we should finally prepare final version 2.0

**Comment by peyre** [comment]
So, are we planning to do our 2.0 release via flatpak?  That's kind of a pain for ordinary users for installing the game.

**Comment by pemensik** [comment]
No, we will make a git tag in original raceintospace repository, and then convert it into a release on that page. This flatpak just follows recent development, because we still do not have a proper stable release. Flatpaks are quite easy to use on Fedora and other distributions. Just Ubuntu pushes its own snap format and therefore does not prepare flatpak repositories in default installation like Fedora does.

**Comment by pemensik** [comment]
We will release ALSO on flatpak. Not only here. No worries. Just making sure we have it ready when we are ready.

**Comment by peyre** [comment]
Thanks Petr!  I was a little nervous about that.

**Comment by pemensik** [comment]
Okay, I will close this issue. I think flatpaks make version out of [doc/org.raceintospace.Raceintospace.metainfo.xml](https://github.com/raceintospace/raceintospace/blob/master/doc/org.raceintospace.Raceintospace.metainfo.xml) in our repository. Displayed version on the other hand in title and credits is generated from src/raceintospace_config.h.in, which gets its value from [CMakeListst.txt](https://github.com/raceintospace/raceintospace/blob/master/doc/org.raceintospace.Raceintospace.metainfo.xml). We have to make it in sync, I don't think we want to generate metainfo versions.

I am trying to ensure modified RC2 candidate in beta branch gets built. It seems to have catched my changes, so it should be ready: https://buildbot.flathub.org/#/builders/32/builds/99929
